doc/source/reference/config.rst

```diff
@@ -65,6 +65,31 @@ If you try to set the default_display to anything other than 'lux' or 'pandas,'
   :width: 700
   :align: center
 
+
+Change the plotting scale 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We can set the :code:`plotting_scale` to scale the size of charts displayed in the Lux widget. In the following block, we set plotting scale to 2, therefore the Lux widget will display charts two times larger than the default. 
```

Comment:
Some suggested changes: 
```
We can adjust the sizes of charts displayed in the Lux widget based on a :code:`plotting_scale` applied to all charts. The default plotting scale is set to 1, displayed as followed: 
[add screenshot of original widget]
In the following block, we set the :code:`plotting_scale` to 2. We see that the Lux widget displays charts two times larger than the default. 
```

Reply:
I suggest that we add the original widget so that we can see the sizes for comparison.